.tests-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+echo "" > coverage.txt
+
+for d in $(go list ./... | grep -v vendor); do
```

Comment:
doesn't works because the package name is `github.com/luizalabs/teresa-api/vendor/github.com/juju`, i.e. both `go list` and `go tests` work with the package name not package dir.